dashboard/app/views/levels/editors/_authored_hints.haml

```diff
@@ -10,7 +10,7 @@
         %input.tts_url{type: 'hidden', value: 'general'}
         .row
           .span10
-            %input.hint_id{type: 'text', placeholder: 'hint_id'}
+            %input.hint_id{type: 'text', placeholder: 'hint_id', required: true}
```

Comment:
Should we update the authored hints without an ID first (or did you already do that)? This might cause issues with people editing levels with hints without an ID.

Or we can just be okay with that and give them a heads up. :)